dbhydra/dbhydra_core.py

```diff
@@ -1287,17 +1281,39 @@ def get_all_columns(self):
 
     def get_all_types(self):
 
+        only_types, lengths = self.get_data_types_and_cahracter_lenghts()
```

Comment:
character, lenGTh

Reply:
two typos in one variable  😆

Reply:
can you describe, what is this function good for? I dont understand + it doesnt have any docstring

Reply:
I added docstring @DovaX